src/components/LoadingSpinnerOverlay.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+    StyleSheet, View, ActivityIndicator,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import themeColors from '../styles/themes/default';
+
+const propTypes = {
+    /** Size of spinner */
+    size: PropTypes.string,
+};
+
+const defaultProps = {
+    size: 'large',
+};
+
+function LoadingSpinnerOverlay(props) {
```

Comment:
Does this component look similar to `FullScreenLoadingIndicator`? Can we utilize the same?

Reply:
It works as expected with `FullScreenLoadingIndicator`.

Reply:
So it means, we can't modify it.

Reply:
I am sorry I missed the `style` props then. We can modify it as `FullScreenLoadingIndicator` accepts a `style` props but the name of this component does not make much sense here. We are showing `FullScreenLoadingIndicator` in place of an `Image`. 

```
<FullscreenLoadingIndicator style={{opacity: 1, backgroundColor: 'transparent'}} />
```

Reply:
Yup. Any suggestions for the name. my mind says `FullAreaLoadingIndicator` | `LoadingIndicator` | `CoverLoadingIndicator`

Reply:
Not sure about it, `FullscreenLoadingIndicator` is a perfect name for what it does. May be we are fine with a separate component or create a generic loader component and create special component with component composition?

Reply:
I am sure we can find a suitable name. @mountiny Do you have a suggestion?